spec/peep_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'peep'
+
+describe Peep do
+
+  describe '#.show' do
```

Comment:
#show is for an instance method, .show is for a class method, so think this should just be .show